src/go/rpk/pkg/config/config_test.go

```diff
@@ -1941,6 +1956,32 @@ func TestReadFlat(t *testing.T) {
 		RequireClientAuth: true,
 	}}
 
+	conf.Redpanda.AdminApi = []NamedSocketAddress{{
+		SocketAddress: SocketAddress{
+			Address: "192.168.92.34",
+			Port:    9644,
+		},
+		Name: "internal",
+	}, {
+		SocketAddress: SocketAddress{
+			Address: "127.0.0.1",
+			Port:    9645,
+		},
+	}}
+
+	conf.Redpanda.AdminApiTLS = []ServerTLS{{
+		Name:     "internal",
+		KeyFile:  "/some/key/file.pem",
+		CertFile: "some/cert/file.crt",
+	}, {
```

Comment:
Is such configuration possible without enable flag set to `true`?

Reply:
The tests doesn't have real configuration. In the test we write the file and read the configuration from memory file system.